docs/scos/dev/feature-walkthroughs/202204.0/click-and-collect-feature-walkthrough/service-point-cart-subdomain-walkthrough.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Click and Collect feature Service Point Cart subdomain walkthrough
+last_updated: Nov 02, 2023
+description: TODO
+template: concept-topic-template
+---
+
+# Service Point Cart
+
+Provides support for service points validation and replacement in a cart.
+
+[Install the Service Points Cart feature](/docs/pbc/all/install-features/{{page.version}}/install-the-service-points-cart-feature.html)
+
+## 1. Modules
+
+| MODULE                      | EXPECTED DIRECTORY                             |
+|-----------------------------|------------------------------------------------|
+| ServicePointCart            | vendor/spryker/service-point-cart              |
+| ServicePointCartExtension   | vendor/spryker/service-point-cart-extension    |
+| ServicePointCartsRestApi    | vendor/spryker/service-point-carts-rest-api    |
+
+## 2. Extension point for item replacement strategy in the cart during checkout process
```

Comment:
* An extension point that allows for the implementation of item replacement strategies in the cart during the checkout process.